### DISCUSSION TITLE
卸载不干净？

### DISCUSSION THREAD

**Discussion by yasuo-lab** [caorushizi/m3u8-downloader#245] [General]
点击卸载程序运行，仍然有相当一部分的玩意残留？你这东西这么恶心？
![Uploading Snipaste_2024-08-09_00-20-08.png…]()
而且安装还没有磁盘选择空间？直接安装c盘？恶心到家了？

**Comment by caorushizi** [comment]
卸载使用的是 electron 自带的卸载工具，这个没办法控制。一键安装在2.2.3中是修改过的，当时图省事选择了一键安装的方式。另外你能不能好好说话。

**Reply by yasuo-lab** [reply]
你要不自己下载试试，你如果觉得不恶心，那我没话可说。

**Reply by ruixingshi** [reply]
图都上传不明白，确实也不知道去官网了解一下更新的road map。我们自己一直在用，所以一直在迭代。
并且我想问问了，你这样说话你觉得恶心吗？你如果觉得不恶心，那我无话可说。
